content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -64,117 +64,55 @@ Open your terminal and navigate to a location of your choice. Run the following
 mkdir prisma-mongodb-cloudflare
 cd prisma-mongodb-cloudflare
 npm init -y
-npm install -D prisma typescript @cloudflare/wrangler webpack
+npm install -D prisma typescript wrangler
 ```
 
 ## 3. Set up Wrangler
 
-Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers.
+Wrangler is the official Cloudflare Worker CLI. You will use it to develop and deploy to Cloudflare Workers. This guide uses [Wrangler v2](https://developers.cloudflare.com/workers/wrangler/).
+
+First, initialize wrangler. To do this, run the following command in your terminal:
 
 ```terminal
 npx wrangler init
 ```
 
-This will create a `wrangler.toml` file with some initial configuration.
-
-Next, authenticate the Wrangler CLI with your Cloudflare Workers account. To do this, run the following command in your terminal:
+This will ask you a few questions.
 
 ```terminal
-npx wrangler login
-Allow Wrangler to open a page in your browser? [y/n] y
+Would you like to use git to manage this Worker? (y/n)
 ```
 
-You can now verify that you're logged in by running `npx wrangler whoami`.
+We want to use git, so answer yes.
```

Comment:
(we seem to capitalize elsewhere, so I'd suggest this even though it's lower case in the command output)

```suggestion
We want to use Git, so answer yes.
```